frb_utils/lib/src/commands/test_web_command.dart

```diff
@@ -24,4 +24,7 @@ class TestWebConfig {
 
   @CliOption(help: 'Make the browser headless', defaultsTo: true)
   late bool headless;
+
+  @CliOption(help: 'Rust feature flags to set during build')
+  late List<String> rustfeatures;
```

Comment:
nit: I would like to merge it, but this indeed should be `rustFeatures`, and the cli will automatically be `rust-features`....